### ISSUE TITLE
Config interface is not exported

### ISSUE THREAD

**Issue by AndyClausen** [tailwindlabs/prettier-plugin-tailwindcss#161]
**What version of `prettier-plugin-tailwindcss` are you using?**
v0.3.0

**What version of Tailwind CSS are you using?**
v3.3.2

**What version of Node.js are you using?**
v18.x

**What package manager are you using?**
npm

**What operating system are you using?**
Windows

**Reproduction URL**
https://replit.com/@AndyClausen/EagerAdvancedExternalcommand#prettier.config.js

**Describe your issue**

Currently, types are not being exported. This means you can't extend your prettier config with the added properties (such as `tailwindAttributes`) without getting type errors.
Here is the error you'll get:
![image](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/assets/16802212/6e825cfe-48fa-4939-868f-eac3d673e3fa)

The interface exists [here](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/blob/cc1b1921d8e9f518d7d15255a0362ca7dba1b197/src/types.d.ts#L29) but is not being exported to dist.

I'd like to be able to do something like this:
```js
/** @type {import("prettier").Config & import("prettier-plugin-tailwindcss")} */
const config = {
  plugins: [require.resolve("prettier-plugin-tailwindcss")],
  tailwindAttributes: ['myClassList'],
};

module.exports = config;
```

A workaround for now is to recreate the interface or make it inline when needed like this:
```js
/** @type {import("prettier").Config & {tailwindAttributes: string[]}} */
```

**Comment by thecrypticace** [comment]
I've merged in #162 which adds exported types for the Prettier config. This will be available in our next tagged release. In the meantime you can test it now by installing our insiders build:
```shell
npm install prettier-plugin-tailwindcss@insiders
```

One note, at least in my setup, a `require()` call is necessary for the prettier config options to show up. Neither the string or `require.resolve` versions of the config will enable typescript to pick up the config. So my config looked like this:

<img width="867" alt="Screenshot 2023-05-19 at 11 50 03" src="https://github.com/tailwindlabs/prettier-plugin-tailwindcss/assets/614993/8d58c473-97c4-4a6e-a9e8-85c613172457">

**Comment by AndyClausen** [comment]
Absolute hero. You can also use resolve if you do this:

```js
/** @type {import("prettier").Config & import("prettier-plugin-tailwindcss").PluginOptions} */
const config = {
  plugins: [require.resolve("prettier-plugin-tailwindcss")],
  tailwindAttributes: ["appearance"],
  tailwindFunctions: [],
};

module.exports = config;
```

But this way, you'll _need_ to add `tailwindAttributes` and `tailwindFunctions` since they're not optional in the interface. Idk if they should be, but just a heads-up :)
